Example-iOS/AppDelegate.swift

```diff
@@ -24,12 +24,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         guard let traitCollection = window?.traitCollection else {
             fatalError("There should be a traitCollection available before calling this method.")
         }
-        let titleStyle = StringStyle.style(
+        let titleStyle = StringStyle(parts:
```

Comment:
#? Why do we even need to have a named param (`parts:`). I feel it's obvious with the `dot` syntax we have going on like `.font().color()`